packages/forms/backend/sirius-components-collaborative-widget-reference/src/main/java/org/eclipse/sirius/components/collaborative/widget/reference/ReferenceWidgetDescriptionConverterSwitch.java

```diff
@@ -268,7 +270,7 @@ private String getDescriptionId(FormElementDescription description) {
         return this.widgetIdProvider.getFormElementDescriptionId(description);
     }
 
-    private IStatus handleItemClick(VariableManager variableManager, List<Operation> operations) {
+    private IStatus newValueHandler(VariableManager variableManager, List<Operation> operations) {
```

Comment:
The error message in this method is no longer coherent with the methode purpose

Reply:
Fixed